modules/serverless-installing-cli-linux.adoc

```diff
@@ -0,0 +1,50 @@
+// Module is included in the following assemblies:
+//
+// serverless/knative-client.adoc
+
+[id="installing-cli-linux_{context}"]
+= Installing `kn` for Linux
+
+For Red Hat Enterprise Linux (RHEL), you can install `kn` as an RPM if you have an active OpenShift Enterprise subscription on your Red Hat account:
+
+[source]
+----
+# subscription-manager register
```

Comment:
I'm unsure as to why these commands are here like this. It looks like there are two ways to install for Linux: as an RPM or as a TAR file. So if you need to provide both options, then the two options should be presented in two different procedures.